fairlearn/postprocessing/_plotting.py

```diff
@@ -132,15 +132,32 @@ def plot_threshold_optimizer(threshold_optimizer: ThresholdOptimizer, ax=None, s
             "$P[\\hat{Y}=1|Y=1]$",
         )
     else:
-        _plot_overall_tradeoff_curve(ax, threshold_optimizer._overall_tradeoff_curve)
-        _plot_solution(
-            ax,
-            threshold_optimizer._x_best,
-            None,
-            "solution",
-            threshold_optimizer.x_metric_,
-            threshold_optimizer.y_metric_,
-        )
+        if hasattr(threshold_optimizer, "_overall_tradeoff_curve"):
```

Comment:
If it has an overall_tradeoff_curve, it means that there is no tolerance, so I use the vanilla plotting function